sonar/translations/manual_translations.txt

```diff
@@ -16,3 +16,17 @@
 # SWITCHaai providers
 _('eduidtest')
 _('eduid')
+
+# Validation workflow status
+_('validation_status_in_progress')
```

Comment:
Why? Where is it used?

Reply:
It is used in frontend. I prefer to keep the maximum translations in the backend.

Reply:
ok